modules/extensions-soap-support/pom.xml

```diff
@@ -28,14 +29,14 @@
         <dependency>
             <groupId>org.mule.runtime</groupId>
             <artifactId>mule-service-soap-api</artifactId>
-            <version>4.0.0-SNAPSHOT</version>
+            <version>${muleSoapServiceApiVersion}</version>
```

Comment:
this is equivalent to project.version. I thought this was referring to the module you added in ext-api.